src/ucs/stats/stats.c

```diff
@@ -46,7 +46,7 @@ enum {
     UCS_ROOT_STATS_LAST
 };
 
-KHASH_MAP_INIT_INT64(ucs_stats_cls, void*)
+KHASH_MAP_INIT_INT64(ucs_stats_cls, ucs_stats_class_t*)
```

Comment:
theoretically, this may not work: if the key is class pointer, and then another SO is reloaded which has another class at exactly the same address..
the key should be class name (which should be unique)
we can use 'char*' as the key (KHASH_MAP_INIT_STR), but need to pass dup->name as the key to kh_put (so it will not be released after SO unload )

Reply:
reload another class into same address? it should be really "lucky" :)
ok, fixed